stderr of test 'remote-table-where-is-null.Bug-6601` in directory 'sql/test/BugTracker-2018` itself:


# 12:25:26 >  
# 12:25:26 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34060" "--set" "mapi_usock=/var/tmp/mtest-30679/.s.monetdb.34060" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-old/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2018" "--set" "embedded_c=true"
# 12:25:26 >  


# 12:25:28 >  
# 12:25:28 >  "Done."
# 12:25:28 >  

